pkg/sql/logictest/testdata/logic_test/jobs

```diff
@@ -13,23 +13,17 @@ GRANT ALL ON DATABASE test TO testuser
 statement ok
 CREATE TABLE t(x INT); INSERT INTO t(x) VALUES (1); CREATE INDEX ON t(x)
 
-# The "updating privileges" clause in the SELECT statement is for excluding jobs
-# run by an unrelated startup migration.
-# TODO (lucy): Update this if/when we decide to change how these jobs queued by
-# the startup migration are handled.
 query TTT
-SELECT job_type, description, user_name FROM [SHOW JOBS] WHERE description != 'updating privileges'
+SELECT job_type, description, user_name FROM [SHOW JOBS]
 ----
-SCHEMA CHANGE  CREATE INDEX ON test.public.t (x)  root
+SCHEMA CHANGE  updating privileges for database 52  root
```

Comment:
did something change to allow the `updating privilege` jobs to show?

Reply:
I guess my question is why did you delete the filter and allow the jobs to show up now

Reply:
That `WHERE` clause would not have filtered out this new job since it used `!=` instead of regex matching. We used to have some unrelated jobs spawned by a startup migration that were (badly) just named `updating privileges`. I could have taken them out months ago when there was a change to bake in that migration. I'm doing it now to make the test less confusing, but it does make the diff very misleading.

Reply:
sounds good